app/views/account/comments/_comment.html.slim

```diff
@@ -0,0 +1,21 @@
+.comment-list[id="#{dom_id(comment)}"]
+  .row
+    .col-sm-1
+      .thumbnail
+        = gravatar_image_tag(current_user.email, gravatar: { size: 50 })
+    .col-sm-11
+      .panel.panel-default
+        .panel-heading
+          strong
+            = link_to comment.user.full_name, account_user_path(comment.user), class: 'user-name'
+          span
+            = simple_format(" commented on #{I18n.l comment.created_at, format: :long}", {}, wrapper_tag: 'small')
```

Comment:
simple_format could not be in span. because it generates "p". FIX HTML

Reply:
PLUS
DO NOT USE SIMPLE FORMAT FOR THIS INFO

IT"S FOR "TEXT" content. Read API about this method